packages/picasso-query-builder/src/QueryBuilder/QueryBuilder.tsx

```diff
@@ -59,23 +59,18 @@ type Props = {
   loading?: boolean
   /** Defines padded layout. */
   padded?: SpacingType
-  /**
-   * @deprecated [CPT-2188] Controls will be defined at the consumer level
-   * Defines the possibility to display, or not, any of the controls. For example "Clear query" or "Run query" control.
-   */
+  /** Defines the possibility to display, or not, any of the controls. For example "Clear query" or "Run query" control. */
   hideControls?: boolean
   /** Defines the possibility to enable, or not, drag-and-drop functionality. This possibility applies to rules and groups to rearrange it within QB. */
   enableDragAndDrop?: boolean
+  /** Defines custom Run Query button children that allows to change button text or display custom logic. */
+  runQueryChildren?: ReactNode
```

Comment:
Nit: runQueryContent as in "the content of run query" seems to be a more expressive option than the "children of run query" wdyt?
```suggestion
  runQueryContent?: ReactNode
```